mgclient_cpp/include/mgclient.hpp

```diff
@@ -248,6 +266,8 @@ inline std::optional<std::vector<std::vector<Value>>> Client::FetchAll() {
   return data;
 }
 
+inline std::vector<std::string> Client::GetColumns() { return columns_; }
```

Comment:
I think a better choice is `const std::vector<std::string>& GetColumns() const {return columns_; }`